spec/helpers/form_error_summary_helper_spec.rb

```diff
@@ -2,8 +2,11 @@
 
 describe FormErrorSummaryHelper do
   describe '#field_anchor_link' do
-    context 'given a field_name' do
-      let(:naming_mock) { double(param_key: 'example_model') }
+    context 'given a model_name and field' do
+      let(:naming_mock) do
+        instance_double(ActiveModel::Name, param_key: 'example_model')
```

Comment:
(I'm a little iffy on this one, but let's talk about that in the context of "the rubocops I happen to dislike" later)